doc/sphinx/practical-tools/coqide.rst

```diff
@@ -5,130 +5,183 @@
+Most editing operations are shown in the *Edit* menu.  Most menu entries show the equivalent
+keystrokes to perform the action.  If you need more complex editor commands, you can
+launch your favorite text editor on the current buffer, using the
+*Edit/External Editor* menu. When you're done editing, you must currently must
+reopen the file to see your changes.  Also note:
+
+- Undo is also available as Ctrl-Z (Command-Z on macOS).  Redo is Ctrl-Shift-Z
+  (Shift-Command-Z on macOS).
+- Select all is Ctrl-A (Command-A on macOS).
+- Ctrl-Delete deletes a word of text after the cursor
+- Ctrl-Backspace deletes a word of text before the cursor
+- Commenting and uncommenting the current line or selected text is available in
+  the *Tools* menu.  If some text is selected, exactly that text is commented out;
+  otherwise the line containing the cursor is commented out.  To uncomment, position
+  the cursor between `(*` and `*)` or select any text between them.
+
+Files are automatically saved periodically to a recovery file.  For example,
+`foo.v` is saved to `#foo.v#` every 10 seconds by default.  You can change the
+interval in the *Edit / Preferences / Files* dialog.
+
+In the *View* menu, you can set several printing options that
+correspond to options that can appear in the script.  For example, "Display
+notations" on the menu corresponds to the :flag:`Printing Notations` flag.  You
+should use the menu instead of controlling these settings in your script.
+
+Running Coq scripts
+-------------------
+
+Operations for running the script are available in the *Navigation* menu,
+from the toolbar and from the keyboard.  These include:
+
+- Forward (Ctrl-Down) to run one command or tactic
+- Backward (Ctrl-Up) undo one command or tactic
+- Run to cursor (Ctrl-Right) to run commands up to the cursor
+- Run to end (Ctrl-End) to run commands to the end of the buffer
+- Reset Coq (Ctrl-Home) to restart the Coq process
+- Interrupt to stop processing commands
+
+Tooltips identify the action assocaited with each toolbar icon.
+
+Commands may:
+
+- Complete successfully.  In this case, the background of the command is marked
+  with the "processed" color (green by default).
+- Complete with a warning.  In this case, the warning appears in the messages
+  panel in yellow.  The background of the command is marked with the "processed"
+  color and the text is shown in blue and underlined.  The message text is
+  available as a tooltip on the text of the command.
+- Fail with an error.  If you're stepping through the proof line by line, the
+  error message appears in the message panel in red and the command is shown
+  in red and underlined with a pink background.  If you're in async mode,
+  described in more detail below, the message appears in the *errors panel*.
+  Double click on an entry to jump to the point of the error.  Execution
+  of commands stops unless you're in async mode.
```

Comment:
Also here, we have Axiom's and Admitted which are highlighted in yellow in the buffer.

Reply:
Think I'd call it light orange.  Different shades in the script and in the ribbon BTW.  Done. but OTOH this is something that users will effortlessly observe (I don't recall ever being told about this behavior).